core/domain/state_domain.py

```diff
@@ -281,8 +281,10 @@ def __init__(
             interaction_id: str. The interaction id.
             answer_is_exclusive: bool. True if is the only correct answer;
                 False if is one of possible answer.
-            correct_answer: str. The correct answer; this answer enables the
-                learner to progress to the next card.
+            correct_answer: list(set) or str. The type of correct_answer is
```

Comment:
Aren't there other possibilities for the type of the correct_answer?

Also, for any composite types, add more detail (e.g. list(set(...))).

Reply:
@seanlip, Actually I realized the correct answers can have many types, like CodeEvaluation(which is a dict), Integer, Real , NumberWithUnits, Graph etc
Should we add all these here? I'm not sure

Reply:
I mentioned this in a couple of other comments already, but you can write `*` to denote that.

Reply:
done